server/remote/gcal/event.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package gcal
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+	msgraph "github.com/yaegashi/msgraph.go/v1.0"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+func (c *client) GetEvent(remoteUserID, eventID string) (*remote.Event, error) {
```

Comment:
Are these not implemented yet, or is there any api limitation not to implement them?

Reply:
@larkox They are just not implemented. I haven't run into anything that seems like a limitation issue on Google's side, but there will probably eventually need to be some changes in the core calendar code to support some feature for both calendars.